qbplatform/quote/models.py

```diff
@@ -41,6 +83,16 @@ class Tossup(models.Model):
 
     packet = models.ForeignKey(Packet, on_delete=models.CASCADE)
 
+    #TU difficulty rating
+    difficulty = models.PositiveSmallIntegerField(null=True)
+
+    #Don't want to cascade delete because distribution might be adjusted after questions are written?
+    #We could lock distribution on tournament creation, but that seems too restrictive
```

Comment:
agree that we should allow editing distributions after the set has been created

Reply:
Yep, same.